### ISSUE TITLE
Not working in Lumen

### ISSUE THREAD

**Issue by pooriamo** [php-telegram-bot/laravel#6]
Hi. I'm getting this error in Lumen:

> Argument 1 passed to PhpTelegramBot\Laravel\PhpTelegramBotServiceProvider::PhpTelegramBot\Laravel\{closure}() must be an instance of Illuminate\Contracts\Foundation\Application, instance of Laravel\Lumen\Application given, called in /vendor/illuminate/container/Container.php on line 776 in /vendor/php-telegram-bot/laravel/src/Laravel/PhpTelegramBotServiceProvider.php:55

**Comment by pooriamo** [comment]
I replaced the type-hint and it's working. But it's a change in the source code. Is there any other solutions for this?

```
/**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
        $this->app->bind(PhpTelegramBotContract::class, function () {
            $app = app();
```

**Comment by akalongman** [comment]
@pooriamo I'll fix that in next release

**Comment by akalongman** [comment]
Fixed in 1.0.1 version
